src/azul/chalice.py

```diff
@@ -191,10 +191,12 @@ def base_url(self) -> furl:
         """
         The base URL of this application. Callers can safely modify the returned
         `furl` instance.
+        The base URL has no path and should be set by caller.
```

Comment:
Empty line between paragraphs.

And this sounds like the caller needs to set the base URL since that's the object in this sentence.

Reply:
Would I instead mention just that there is no path set for the base URL (`The base URL has no path.`)? or would `Path has not been set.` work? (not mentioning `base URL` due to it seeming repetitive)